Automattic/jetpack
# Boost: Refetch no-boost scores if theme changed

#### Changes proposed in this Pull Request:
Boost now fetches noBoost score again even within the cache time if the theme was changed.

#### Jetpack product discussion
p1633516270014500/1633441358.008100-slack-C0232B8CUHG

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* Take boost scores through the boost option page.
* Change the active theme of the site.
* Take boost scores again.
* See if the noBoost score is updated.
